scripts/release-tools/kube-burner-configs/cluster-density/config.yml

```diff
@@ -11,7 +11,7 @@ global:
 jobs:
   - name: cluster-density
     namespace: cluster-density
-    jobIterations: 10
+    jobIterations: 5
```

Comment:
There are now two types of deployments so cut the number of jobIterations in half to keep the same number of deployments.